node-repository/src/test/java/com/yahoo/vespa/hosted/provision/provisioning/AllocationSimulator.java

```diff
@@ -11,12 +11,8 @@
 import com.yahoo.vespa.hosted.provision.NodeList;
 import com.yahoo.vespa.hosted.provision.node.Allocation;
 import com.yahoo.vespa.hosted.provision.node.Generation;
-import com.yahoo.vespa.hosted.provision.node.History;
-import com.yahoo.vespa.hosted.provision.node.IP;
-import com.yahoo.vespa.hosted.provision.node.Reports;
-import com.yahoo.vespa.hosted.provision.node.Status;
 
-import javax.swing.JFrame;
+import javax.swing.*;
```

Comment:
```suggestion
import javax.swing.JFrame;
```

Reply:
Thought I already disabled wildcard imports, but apparently there was a default exception to `java.awt.*` and `java.swing.*` in Intellij